docs/root/configuration/listener_filters/proxy_protocol.rst

```diff
@@ -0,0 +1,17 @@
+.. _config_listener_filters_proxy_protocol:
+
+Proxy Protocol
+==============
+
+This listener filter adds support for
+`HAProxy Proxy Protocol <http://www.haproxy.org/download/1.9/doc/proxy-protocol.txt>`_.
```

Comment:
i know it's in the code comments, but let's make it abundantly clear in the public docs. the filter supports both v1 and v2 simultaneously and automatically determines which version is being sent. rough explanation of error handling behavior, i.e. it throws an error and stops the request if it can't parse the protocol, etc. also call out the lack of AF_UNIX support.